### PR TITLE
Rewrite css/cssom-1/index-002 from generate_tests syntax

### PR DESCRIPTION
**Pull request by nearestnabors**
In an effort to help @gsnedders bring the CSS tests into the fold with the current syntax, this PR removes the `generate_tests()` syntax from `css/cssom-1/index-002.html` and renames the file to what it's actually testing for, shorthand values. There are perhaps additional updates that can be made to these tests, but the main goal of this PR is to remove `generate_tests()`.

### CODE REVIEW

File: css/cssom-1/shorthand-values.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        {name: 'list', setCSS: 'list-style-type: lower-alpha;', expected: 'list-style-type: lower-alpha;'},
+        {name: 'list', setCSS: 'list-style-type: circle; list-style-position: inside; list-style-image: initial;', expected: 'list-style: circle inside;'},
+        {name: 'font-family', setCSS: 'font-family: sans-serif; line-height: 2em; font-size: 3em; font-style: italic; font-weight: bold;', expected: 'font-family: sans-serif; line-height: 2em; font-size: 3em; font-style: italic; font-weight: bold;'},
+        {name: 'padding', setCSS: 'padding-top: 1px; padding-right: 2px; padding-bottom: 3px; padding-left: 4px;', expected: 'padding: 1px 2px 3px 4px;'}
+      ];
+
+      var testElm = document.getElementById('cssomtestElm');
+
+      tests.forEach(function (testcase) {
+        testElm.setAttribute('style', testcase.setCSS);
```
Comment (gsnedders) [inline css/cssom-1/shorthand-values.html]: All of this should be moved within the test call.

**Comment by gsnedders** [comment]
Aside from that, this looks great! Thanks @rachelnabors!

I wonder how much sense it makes to have the expected value in the test name, as there are multiple inputs that could result in the same expected value, and we can't have duplicate test names. Maybe worth changing that to the input?

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/218240359)

# Firefox (nightly channel) #
Testing web-platform-tests at revision b9bd705a21d4abaebc1a48f1c2e28f18baeca79e
Using browser at version BuildID 20170403100136; SourceStamp 38894655c89e68bcd8f45d31a0d3005f2c2b53db
Starting 10 test iterations
## Unstable results ##


|                 Test                 |                  Subtest                   |     Results     |                                                                                         Messages                                                                                         |
|--------------------------------------|--------------------------------------------|-----------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| `/css/cssom-1/shorthand-values.html` | `border was expected to be "border: 1px;"` | **FAIL: 20/10** | `assert_equals: expected "border: 1px;" but got "border: 1px none;";assert_equals: expected "border: 1px;" but got "border-width: 1px; border-style: none; border-color: currentcolor;"` |

## All results ##
<details>
<summary>2 tests ran</summary>
<details>
<summary><a href="http://w3c-test.org/submissions/5331/IndexedDB/parallel-cursors-upgrade.html">/IndexedDB/parallel-cursors-upgrade.html</a></summary>


|     Subtest     | Results | Messages |
|-----------------|---------|----------|
|                 | OK      |          |
| `2 cursors`     | PASS    |          |
| `10 cursors`    | PASS    |          |
| `100 cursors`   | PASS    |          |
| `1000 cursors`  | PASS    |          |
| `10000 cursors` | PASS    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5331/css/cssom-1/shorthand-values.html">/css/cssom-1/shorthand-values.html</a></summary>


|                                                                  Subtest                                                                   |     Results     |                                                                                                                                                                                                    Messages                                                                                                                                                                                                   |
|--------------------------------------------------------------------------------------------------------------------------------------------|-----------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
|                                                                                                                                            | ERROR           |                                                                                                                                                                                                                                                                                                                                                                                                               |
| `border was expected to be "border: 1px;"`                                                                                                 | **FAIL: 20/10** | `assert_equals: expected "border: 1px;" but got "border: 1px none;";assert_equals: expected "border: 1px;" but got "border-width: 1px; border-style: none; border-color: currentcolor;"`                                                                                                                                                                                                                      |
| `border was expected to be "border: 1px solid red;"`                                                                                       | PASS            |                                                                                                                                                                                                                                                                                                                                                                                                               |
| `border was expected to be "border: 1px red;"`                                                                                             | FAIL            | `assert_equals: expected "border: 1px red;" but got "border: 1px none red;"`                                                                                                                                                                                                                                                                                                                                  |
| `border was expected to be "border: red;"`                                                                                                 | FAIL            | `assert_equals: expected "border: red;" but got "border: medium none red;"`                                                                                                                                                                                                                                                                                                                                   |
| `border was expected to be "border-width: 1px 2px 3px 4px;"`                                                                               | FAIL            | `assert_equals: expected "border-width: 1px 2px 3px 4px;" but got "border-width: 1px 2px 3px 4px; border-style: none; border-color: currentcolor;"`                                                                                                                                                                                                                                                           |
| `border was expected to be "border-width: 1px;"`                                                                                           | PASS            |                                                                                                                                                                                                                                                                                                                                                                                                               |
| `border was expected to be "border: dotted;"`                                                                                              | FAIL            | `assert_equals: expected "border: dotted;" but got "border: medium dotted;"`                                                                                                                                                                                                                                                                                                                                  |
| `border was expected to be "border-width: 1px; border-top-color: red;"`                                                                    | FAIL            | `assert_equals: expected "border-width: 1px; border-top-color: red;" but got "border-width: 1px; border-style: none; border-color: red currentcolor currentcolor; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none;"`                                                                                         |
| `border was expected to be "border-right-width: 1px; border-bottom-width: 1px; border-left-width: 1px; border-top-width: 1px !important;"` | FAIL            | `assert_equals: expected "border-right-width: 1px; border-bottom-width: 1px; border-left-width: 1px; border-top-width: 1px !important;" but got "border-right: 1px none; border-bottom: 1px none; border-left: 1px none; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none; border-top: 1px none !important;"` |
| `border was expected to be "border-width: 2px 1px 1px;"`                                                                                   | FAIL            | `assert_equals: expected "border-width: 2px 1px 1px;" but got "border-width: 2px 1px 1px; border-style: none; border-color: currentcolor; -moz-border-top-colors: none; -moz-border-right-colors: none; -moz-border-bottom-colors: none; -moz-border-left-colors: none; border-image: none;"`                                                                                                                 |
| `overflow was expected to be "overflow: scroll hidden;"`                                                                                   | FAIL            | `assert_equals: expected "overflow: scroll hidden;" but got "overflow-x: scroll; overflow-y: hidden;"`                                                                                                                                                                                                                                                                                                        |
| `overflow was expected to be "overflow: scroll;"`                                                                                          | PASS            |                                                                                                                                                                                                                                                                                                                                                                                                               |
| `outline was expected to be "outline: blue dotted 2px;"`                                                                                   | FAIL            | `assert_equals: expected "outline: blue dotted 2px;" but got "outline: 2px dotted blue;"`                                                                                                                                                                                                                                                                                                                     |
| `margin was expected to be "margin: 1px 2px 3px 4px;"`                                                                                     | PASS            |                                                                                                                                                                                                                                                                                                                                                                                                               |
| `list was expected to be "list-style-type: lower-alpha;"`                                                                                  | PASS            |                                                                                                                                                                                                                                                                                                                                                                                                               |
| `list was expected to be "list-style: circle inside;"`                                                                                     | FAIL            | `assert_equals: expected "list-style: circle inside;" but got "list-style-type: circle; list-style-position: inside; list-style-image: initial;"`                                                                                                                                                                                                                                                             |
| `font-family was expected to be "font-family: sans-serif; line-height: 2em; font-size: 3em; font-style: italic; font-weight: bold;"`       | PASS            |                                                                                                                                                                                                                                                                                                                                                                                                               |
| `padding was expected to be "padding: 1px 2px 3px 4px;"`                                                                                   | PASS            |                                                                                                                                                                                                                                                                                                                                                                                                               |

</details>
</details>

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/218240360)

# Chrome (unstable channel) #
Testing web-platform-tests at revision b9bd705a21d4abaebc1a48f1c2e28f18baeca79e
Using browser at version 59.0.3053.3 dev
Starting 10 test iterations
## Unstable results ##


|                 Test                 |                  Subtest                   |    Results     |                                                       Messages                                                      |
|--------------------------------------|--------------------------------------------|----------------|---------------------------------------------------------------------------------------------------------------------|
| `/css/cssom-1/shorthand-values.html` | `border was expected to be "border: 1px;"` | **FAIL, PASS** | `assert_equals: expected "border: 1px;" but got "border-width: 1px; border-style: initial; border-color: initial;"` |

## All results ##
<details>
<summary>2 tests ran</summary>
<details>
<summary><a href="http://w3c-test.org/submissions/5331/IndexedDB/parallel-cursors-upgrade.html">/IndexedDB/parallel-cursors-upgrade.html</a></summary>


|     Subtest     | Results | Messages |
|-----------------|---------|----------|
|                 | OK      |          |
| `2 cursors`     | PASS    |          |
| `10 cursors`    | PASS    |          |
| `100 cursors`   | PASS    |          |
| `1000 cursors`  | PASS    |          |
| `10000 cursors` | PASS    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5331/css/cssom-1/shorthand-values.html">/css/cssom-1/shorthand-values.html</a></summary>


|                                                                  Subtest                                                                   |    Results     |                                                                                                                            Messages                                                                                                                            |
|--------------------------------------------------------------------------------------------------------------------------------------------|----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
|                                                                                                                                            | ERROR          |                                                                                                                                                                                                                                                                |
| `border was expected to be "border: 1px;"`                                                                                                 | **FAIL, PASS** | `assert_equals: expected "border: 1px;" but got "border-width: 1px; border-style: initial; border-color: initial;"`                                                                                                                                            |
| `border was expected to be "border: 1px solid red;"`                                                                                       | PASS           |                                                                                                                                                                                                                                                                |
| `border was expected to be "border: 1px red;"`                                                                                             | PASS           |                                                                                                                                                                                                                                                                |
| `border was expected to be "border: red;"`                                                                                                 | PASS           |                                                                                                                                                                                                                                                                |
| `border was expected to be "border-width: 1px 2px 3px 4px;"`                                                                               | FAIL           | `assert_equals: expected "border-width: 1px 2px 3px 4px;" but got "border-width: 1px 2px 3px 4px; border-style: initial; border-color: initial;"`                                                                                                              |
| `border was expected to be "border-width: 1px;"`                                                                                           | PASS           |                                                                                                                                                                                                                                                                |
| `border was expected to be "border: dotted;"`                                                                                              | PASS           |                                                                                                                                                                                                                                                                |
| `border was expected to be "border-width: 1px; border-top-color: red;"`                                                                    | FAIL           | `assert_equals: expected "border-width: 1px; border-top-color: red;" but got "border-width: 1px; border-style: initial; border-right-color: initial; border-bottom-color: initial; border-left-color: initial; border-image: initial; border-top-color: red;"` |
| `border was expected to be "border-right-width: 1px; border-bottom-width: 1px; border-left-width: 1px; border-top-width: 1px !important;"` | FAIL           | `assert_equals: expected "border-right-width: 1px; border-bottom-width: 1px; border-left-width: 1px; border-top-width: 1px !important;" but got "border-right: 1px; border-bottom: 1px; border-left: 1px; border-image: initial; border-top: 1px !important;"` |
| `border was expected to be "border-width: 2px 1px 1px;"`                                                                                   | FAIL           | `assert_equals: expected "border-width: 2px 1px 1px;" but got "border-width: 2px 1px 1px; border-style: initial; border-color: initial; border-image: initial;"`                                                                                               |
| `overflow was expected to be "overflow: scroll hidden;"`                                                                                   | FAIL           | `assert_equals: expected "overflow: scroll hidden;" but got "overflow-x: scroll; overflow-y: hidden;"`                                                                                                                                                         |
| `overflow was expected to be "overflow: scroll;"`                                                                                          | PASS           |                                                                                                                                                                                                                                                                |
| `outline was expected to be "outline: blue dotted 2px;"`                                                                                   | PASS           |                                                                                                                                                                                                                                                                |
| `margin was expected to be "margin: 1px 2px 3px 4px;"`                                                                                     | PASS           |                                                                                                                                                                                                                                                                |
| `list was expected to be "list-style-type: lower-alpha;"`                                                                                  | PASS           |                                                                                                                                                                                                                                                                |
| `list was expected to be "list-style: circle inside;"`                                                                                     | PASS           |                                                                                                                                                                                                                                                                |
| `font-family was expected to be "font-family: sans-serif; line-height: 2em; font-size: 3em; font-style: italic; font-weight: bold;"`       | PASS           |                                                                                                                                                                                                                                                                |
| `padding was expected to be "padding: 1px 2px 3px 4px;"`                                                                                   | PASS           |                                                                                                                                                                                                                                                                |

</details>
</details>

**Comment by w3c-bots** [comment]
These tests are now available on [w3c-test.org](https://w3c-test.org/submissions/5331/)

**Comment by gsnedders** [comment]
Oh, huh, per the stability checker we genuinely have a duplicate test name in there, so my previous comment wasn't a hypothetical concern.

As for authors, we should entirely have that in git hence we don't need them in the document (and we can remove ones which already appear in the file's history, which I suspect is the case here without checking).

File: css/cssom-1/shorthand-values.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        var specifiedValue = testElm.style.cssText;
+        var expectedText = testcase.name + ' was expected to be "' + testcase.expected + '"';
+          test( function () {
+              assert_equals(specifiedValue, testcase.expected);
+          }, expectedText);
```
Comment (zcorpan) [inline css/cssom-1/shorthand-values.html]: Remove `expectedText` and build a test name from `testcase.name` and `testcase.setCSS` instead.

**Comment by zcorpan** [comment]
I suppose this now needs a rebase to move to /cssom/

**Comment by gsnedders** [comment]
@rachelnabors do you have any plan to pick this up again or should someone else take this over?

**Pull request closed without merging**